rust/parser/typeql.pest

```diff
@@ -198,7 +198,7 @@ definable = { ( definition_type ~ SEMICOLON ) | definition_function | definition
 
 // TYPE DEFINITION =============================================================
 
-definition_type = { kind? ~ label ~ ( type_capability ~ ( COMMA ~ type_capability )* )? }
+definition_type = { kind? ~ label ~ (annotations | type_capability)? ~ ( COMMA ~ type_capability )* }
```

Comment:
```suggestion
definition_type = { kind? ~ label ~ ( ( annotations | type_capability ) ~ ( COMMA ~ type_capability )* )? }
```

Reply:
Will do. But my original formulation (and this one) has a drawback that there's no clean standard formatting anymore.
```
relation relates first-role,
    relates second-role;
````

Thoughts?

Reply:
Cleared up in person, a small confusion about the grammar.